ui/components/component-library/modal-body/__snapshots__/modal-overlay.test.tsx.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ModalBody should match snapshot 1`] = `
+<div>
+  <div
+    aria-hidden="true"
```

Comment:
Remove aria-hidden and update test to have children. Check if ModalBody should have any special sematic html or accessibility roles